web/src/main/java/org/phoenixctms/ctsms/web/jersey/resource/shared/SearchResource.java

```diff
@@ -312,10 +314,10 @@ public ResourceIndex index(@Context Application application,
 	public Page<NoShortcutSerializationWrapper<UserOutVO>> searchUser(CriterionInVOWrapper criteria, @Context UriInfo uriInfo)
 			throws AuthenticationException, AuthorisationException, ServiceException {
 		PSFUriPart psf = new PSFUriPart(uriInfo);
-		Collection result = WebUtil.getServiceLocator().getSearchService()
+		Collection<UserOutVO> result = WebUtil.getServiceLocator().getSearchService()
 				.searchUser(auth, criteria, new LinkedHashSet<CriterionInVO>(criteria.getCriterions()), ResourceUtils.LIST_GRAPH_MAX_USER_INSTANCES, psf);
-		NoShortcutSerializationWrapper.transformVoCollection(result);
-		return new Page<NoShortcutSerializationWrapper<UserOutVO>>(result, psf);
+		List<NoShortcutSerializationWrapper<UserOutVO>> transformedResult = result.stream().map(x -> new NoShortcutSerializationWrapper<UserOutVO>(x)).collect(Collectors.toList());
```

Comment:
makes only sense, if this pattern is changed everywhere in a uniform way. massive in ctsms-core, also from andromda templates; low benefit.

btw: in general it must be ensured, that the collection transformation are not processed in parallel (see core.vocycle package)!

Reply:
I've noticed the heavy use in core, but the change at least reduced on direct dependency in `web` and removed a couple of lines of code.
For the sake of uniformity it might be better to remove this part of the PR

Reply:
"transform" pattern was not my favour either, but continued it for conformity. unless not absolutely mandatory (eg. because jersey cant be updated otherwise) i would agree to keep it as is for now.

Reply:
No, that was just an itch. Jersey doesn't bother.